scripts/teahouse/send_th_invites.py

```diff
@@ -57,37 +56,32 @@ def inviteGuests(prof, inviter):
     Invites todays newcomers.
     """
     if inviter:
-        message = hb_config.message.format(inviter=inviter).encode('utf-8')
+        prof.invite = hb_config.message.format(inviter=inviter).encode('utf-8')
     else:
-        message = hb_config.message.encode('utf-8')
-    try:
-        prof.getToken()
-        prof.publishProfile()
-    except:
-        print "something went wrong trying to invite " + page_path
+        # Calling format even if there is no inviter field, to always
+        # have the same amount of curly brackets in the message.
+        prof.invite = hb_config.message.format().encode('utf-8')
+    prof.getToken()
+    prof.publishProfile()
     return prof
 
 if __name__ == "__main__":
-    param = hb_output_settings.Params()
-    params = param.getParams(sys.argv[1])
     # params contains information about skipped templates, user talk
     # namespace and the query for getting candidates below.
-    elig_check = hb_toolkit.Eligible(params)
+    elig_check = hb_toolkit.Eligible()
 
     daily_sample = hb_profiles.Samples()
     daily_sample.insertInvitees("teahouse experiment newbies") #need to generalize for TWA too
     daily_sample.updateTalkPages("th add talkpage") #need to generalize for TWA too
-    candidates = daily_sample.selectSample(params['select query'], sub_sample=False)
+    select_query = "th experiment invitees"
```

Comment:
Considering we have hard coded the other two query references ("teahouse experiment newbies" and "th add talkpage" we can jsut stick this string straight into the selectSample call.